rust/src/lib.rs

```diff
@@ -113,3 +119,26 @@ impl S3DataKeys {
             .collect()
     }
 }
+
+#[inline]
+#[must_use]
+/// Return AWS SDK config with optional region name to use.
+pub async fn get_aws_config(region: Option<String>) -> SdkConfig {
+    aws_config::from_env()
+        .region(get_region_provider(region))
+        .load()
+        .await
+}
+
+#[inline]
+#[must_use]
+/// Return new AWS STS client.
+pub fn aws_sts_client(config: &SdkConfig) -> aws_sdk_sts::Client {
+    aws_sdk_sts::Client::new(config)
+}
```

Comment:
How come we are writing such trivial functions instead of just calling this directly? Seems unnecessary, especially for the STS client 🤔

Reply:
This way I could hide the aws import from the cli side so you don't have to import it since that module does not otherwise use any aws crates. Now it is not maybe so needed since the cli is part of the library too.

Reply:
Ah okay, although preventing the import statement doesn't change the fact that the CLI is using the aws_sdk structs, so don't know exactly how that works 🤔  🤷